contrib-02-lhcb-openstack/Readme.md

```diff
@@ -0,0 +1,192 @@
+| OS-EXT-STS:power_state               | 0                                               |
+| OS-EXT-STS:task_state                | scheduling                                      |
+| OS-EXT-STS:vm_state                  | building                                        |
+| OS-SRV-USG:launched_at               | -                                               |
+| OS-SRV-USG:terminated_at             | -                                               |
+| accessIPv4                           |                                                 |
+| accessIPv6                           |                                                 |
+| config_drive                         |                                                 |
+| created                              | 2016-03-16T13:25:41Z                            |
+| flavor                               | m2.large (12076)                                |
+| hostId                               |                                                 |
+| id                                   | 335994c0-7e06-4423-afd3-5a21836b0cca            |
+| image                                | CernVM 3 (3d46fbda-97ba-4c8b-a48b-f2e3c3e418b3) |
+| key_name                             | CERN                                            |
+| metadata                             | {}                                              |
+| name                                 | valgrinder                                      |
+| os-extended-volumes:volumes_attached | []                                              |
+| progress                             | 0                                               |
+| security_groups                      | default                                         |
+| status                               | BUILD                                           |
+| tenant_id                            | 921dfe9a-9396-4d1b-aa9d-9c7ebd963235            |
+| updated                              | 2016-03-16T13:25:41Z                            |
+| user_id                              | apuignav                                        |
++--------------------------------------+-------------------------------------------------+
+```
+
+
+It takes about 10 minutes until the machine can be reached via ssh. 
+You can query the status of the instance via  
+```bash
+nova show <host-name>
+```
+
+As soon networking is done and the machine has booted, you can log in via
+
+```
+ssh default@<host-name>
+```
+The `default` user has password `testtest`. You should change it after you log in.
+
+You can also log in as root, but the LHCbSoftware will not be available
+```
+ssh root@<host-name>
+```
+The key which you specified in the `nova boot` command is the key which is used by `ssh` for authentication as the `root` user.
+
+
+### Set up AFS
+In order to use afs, you need to authenticate with keberos after login:
+```
+kinit <your-cern-username>@CERN.CH
+aklog
+```
```

Comment:
Well, any note needs some conclusions/closure. Summarize what you have presented, maybe say that there is an automated way of doing it (and give a link) and remind the reader why this is important. It's just a bit of blabla, but a lot of people just read the conclusions and then go back to the article if they like what they read.